README-zh_CN.md

```diff
@@ -56,6 +56,7 @@ Lunettes 可以识别容器生命周期每个交付阶段的开始和结尾，
 # install lunettes
 # 该过程开启apiserver的审计功能，apiserver会发生重启
 helm upgrade --install lunettes oci://registry-1.docker.io/lunettes/lunettes-chart --version [version] 
+
```

Comment:
Please remove useless blank lines to avoid conflicts with other pr merges

Reply:
> Please remove useless blank lines to avoid conflicts with other pr merges

done